src/Migrator.EF6.Tools/Executor.cs

```diff
@@ -67,7 +67,11 @@ public void EnableMigrations(string outputDir)
 
 		private string FindAppDbContextTypeName()
 		{
-			var allDbContextTypes = _types.Where(t => typeof(DbContext).IsAssignableFrom(t));
+			var allDbContextTypes = _types.Where(
+				t => typeof(DbContext).IsAssignableFrom(t)
```

Comment:
Can we make this a method and use it in `FindDbMigrationsConfiguration` too? (it's duplicated)
Perhaps `var allDbContextTypes = GetConstructablesOfType(typeof(DbContext))`.